lib/resty/dns/resolver.lua

```diff
@@ -41,6 +41,7 @@ local IP6_ARPA = "ip6.arpa"
 local TYPE_A      = 1
 local TYPE_NS     = 2
 local TYPE_CNAME  = 5
+local TYPE_SOA    = 6
```

Comment:
I think you should add the new constant to the documentation (`README.md`) accordingly? See

https://github.com/openresty/lua-resty-dns/#constants